src/python_testing/TC_RR_1_1.py

```diff
@@ -351,6 +355,57 @@ async def test_TC_RR_1_1(self):
         else:
             logging.info("Step 9: Skipped due to no UserLabel cluster instances")
 
+        # Step 10: Count all group cluster instances
+        # and ensure MaxGroupsPerFabric >= 4 * counted_groups_clusters.
+        logging.info("Step 10: Validating groups support minimums")
+        groups_cluster_endpoints: Dict[int, Any] = await dev_ctrl.ReadAttribute(self.dut_node_id, [Clusters.Groups])
+        counted_groups_clusters: int = len(groups_cluster_endpoints)
+
+        # The test for Step 10 and all of Steps 11 to 14 are only performed if Groups cluster instances are found.
+        if counted_groups_clusters > 0:
+            indicated_max_groups_per_fabric: int = await self.read_single_attribute(dev_ctrl,
+                                                                                    node_id=self.dut_node_id,
+                                                                                    endpoint=0,
+                                                                                    attribute=Clusters.GroupKeyManagement.Attributes.MaxGroupsPerFabric)
+            if indicated_max_groups_per_fabric < 4 * counted_groups_clusters:
```

Comment:
I believe this should just verify if indicated_max_groups_per_fabric < 4 : then assert.fail(f"Failed Step Step 10: MaxGroupsPerFabric < 4")

Reply:
Based on the updated spec text, group support must scale with the number of endpoints supporting groups.

> The node SHALL support at least four groups per fabric per endpoint having a Groups cluster instance.